content/integrations/blog.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "OpenCensus for IoT"
+date: 2018-08-02
+draft: false
+weight: 3
+class: "resized-logo"
+---
+
+![](/images/IoT-GoogleCloud.jpg)
+
+## Table of contents
+- [Background](#background)
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [Packages to import](#packages-to-import)
+- [Technical detour](#technical-detour)
+  - [System I](#system-i)
+  - [System II](#system-ii)
+    - [Protocol](#protocol)
+- [Reference](#reference)
+- [Acknowledgement](#acknowledgement)
+
+#### Background
+
+Our application is a prototype which demonstrates how we can utilize OpenCensus for the observability in the IoT industry.  With the service of metric collection and monitoring provided by OpenCensus, we'll gain the insights into hardware and software performance across distributed IoT devices.
+
+#### Requirements
+
+* Raspberry Pi 2B+ with Go 1.8 and greater installed
+* PC under Mac OS or Ubuntu
+* Raspberry Pi and PC are under the same LAN
+* Arduino UNO with Arduino IDE 1.8.5 and greater
+* Sensors including DHT11, HTU21D and SparkFun sound detector
+* Enable Stackdriver Monitoring on the Project
```

Comment:
Consider changing this to "A Google Cloud project with Stackdriver Monitoring enabled". You can also mention that we may extend the prototype to support other metric backends, like Prometheus.